docs/docs/guides/18_resources_and_troubleshooting/index.md

```diff
@@ -44,6 +44,24 @@ Additional Info:
 
 [Facebook/React-native Issue #28492](https://github.com/facebook/react-native/issues/28492#issuecomment-824698934)
 
+### TypeError: Cannot read property 'prototype' of undefined, js engine: hermes
+
+This error occurs when trying to use Web3.js with React Native. To solve this error, use [the `react-native-quick-crypto` package](https://www.npmjs.com/package/react-native-quick-crypto).
```

Comment:
@danforbes thanks, this is great! maybe also add mention that web3 has to be imported as a default import, as named import doesn't work. I tried to explain it here https://github.com/web3/web3.js/issues/7313#issuecomment-2486142506

```
import Web3 from 'web3';
```